app/src/main/res/layout/hints_summary.xml

```diff
@@ -19,23 +19,24 @@
     android:id="@+id/hint_summary_container"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_marginTop="32dp"
+    android:layout_marginTop="@dimen/hints_summary_linear_layout_margin_top"
     android:orientation="vertical">
 
     <androidx.constraintlayout.widget.ConstraintLayout
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:layout_marginStart="28dp"
-      android:layout_marginEnd="28dp"
-      android:layout_marginBottom="20dp">
+      android:layout_marginStart="@dimen/hints_summary_constraint_layout_layout_margin_start"
+      android:layout_marginEnd="@dimen/hints_summary_constraint_layout_layout_margin_end"
+      android:layout_marginBottom="@dimen/hints_summary_constraint_layout_layout_margin_bottom">
 
       <TextView
         android:id="@+id/hint_title"
         style="@style/TextViewStart"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginTop="12dp"
-        android:layout_marginBottom="12dp"
+        android:layout_marginTop="@dimen/hint_title_margin_top"
+        android:layout_marginBottom="@dimen/hint_title_margin_bottom"
+        android:paddingBottom="@dimen/hint_title_padding_bottom"
```

Comment:
Here, and in other places also you have to change the dimen name exactly same as you mentioned in dimens.xml. Here, you have written `@dimen/hint_title_padding_bottom` which should be `@dimen/hints_summary_hint_title_padding_bottom`